platform/ui/src/components/downloadDialog/DownloadDialog.js

```diff
@@ -75,24 +76,31 @@ const DownloadDialog = ({
   }, [defaultSize, disableViewport, enableViewport, viewportElement]);
 
   useEffect(() => {
+    const getValidMeasure = value => {
```

Comment:
I don't follow why this needs to be wrapped in `useEffect`. Any of the `set` calls will trigger a re-render. You can have this be a normal func declared beneath `DownloadDialog`'s function.

Reply:
This is because the maximumSize and minimumSize properties are being used to process the valid measure, the useEffect have those being watched. I have simplified this function though since im clamping the value in the onChange for max values. 

Done!, thx.